lib/cog/adapter.ex

```diff
@@ -45,7 +45,8 @@ defmodule Cog.Adapter do
       end
 
       defp payload(sender, room, text) do
-        %{sender: sender,
+        %{id: UUID.uuid4(:hex),
```

Comment:
Any reason to use `:hex` value here? The `:default` version might be nice if we were want to store these in postgres.